src/utils/api.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+export const postLogin = (email, password) => {
+    return axios.post(
+        'http://localhost:5433/user/login',
```

Comment:
We should come up with a way to dynamically manage the port. The default from the API is `3000`, but in your case, it's `5433`, so we shouldn' t hardcode this. Possibly `.env`?

Reply:
Cool, I was thinking .env